app/controllers/product_access_groups_controller.rb

```diff
@@ -50,4 +50,14 @@ def destroy
     end
   end
 
+  private
+
+  def create_params
```

Comment:
Since create/update are identical, should they be just one method?

Reply:
cancancan looks for the methods to match the controller action names, so I think we need both.